<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GENFIRE_MainWindow</class>
 <widget class="QMainWindow" name="GENFIRE_MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1093</width>
    <height>876</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>GENFIRE</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-181</y>
         <width>1039</width>
         <height>1012</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="nameWidget" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>512</width>
              <height>93</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-image: url(:/GENFIRE.png)</string>
            </property>
            <zorder></zorder>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="9" column="2">
             <widget class="QLineEdit" name="lineEdit_io"/>
            </item>
            <item row="10" column="0" colspan="3">
             <widget class="Line" name="line_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLineEdit" name="lineEdit_pj"/>
            </item>
            <item row="12" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Results Filename</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="3">
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="7" column="0" colspan="3">
             <widget class="Line" name="line_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="13" column="2">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>*.mrc, *.npy, or *.mat</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item row="5" column="0" colspan="3">
             <widget class="Line" name="line_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="12" column="2">
             <widget class="QLineEdit" name="lineEdit_results"/>
            </item>
            <item row="3" column="0">
             <widget class="QPushButton" name="btn_support">
              <property name="text">
               <string>Select Support File</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QPushButton" name="btn_angles">
              <property name="text">
               <string>Select Angle File</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QCheckBox" name="checkBox_default_support">
              <property name="text">
               <string>Use default support</string>
              </property>
             </widget>
            </item>
            <item row="13" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout"/>
            </item>
            <item row="3" column="2">
             <widget class="QLineEdit" name="lineEdit_support"/>
            </item>
            <item row="9" column="0">
             <widget class="QPushButton" name="btn_io">
              <property name="text">
               <string>Select Inital Object File</string>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QCheckBox" name="checkBox_provide_io">
              <property name="text">
               <string>Provide initial object</string>
              </property>
             </widget>
            </item>
            <item row="6" column="2">
             <widget class="QLineEdit" name="lineEdit_angle"/>
            </item>
            <item row="0" column="0">
             <widget class="QPushButton" name="btn_projections">
              <property name="text">
               <string>Select Projection File</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>Resolution Extension/Suppression</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout">
               <item>
                <widget class="QRadioButton" name="radioButton_on">
                 <property name="text">
                  <string>On</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_off">
                 <property name="text">
                  <string>Off</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_extension">
                 <property name="text">
                  <string>Extension Only</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>Method</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QRadioButton" name="radioButton_ER">
                 <property name="text">
                  <string>ER</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioButton_DR">
                 <property name="text">
                  <string>DR</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="3" column="0" colspan="2">
               <widget class="QCheckBox" name="checkBox_multiGridding">
                <property name="text">
                 <string>Permit Multiple Grid-Point Matches</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="Line" name="line_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="4" column="3">
               <widget class="QCheckBox" name="checkBox_rfree">
                <property name="text">
                 <string>Calculate Rfree</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_7">
                <property name="text">
                 <string>Gridding Method</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0" colspan="2">
               <widget class="QCheckBox" name="checkBox_resCircle">
                <property name="text">
                 <string>Enforce Resolution Circle</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="3">
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Constraints</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignCenter</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_positivity_constraint">
                  <property name="text">
                   <string>Positivity</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_support_constraint">
                  <property name="text">
                   <string>Support</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="6" column="3">
               <widget class="QPushButton" name="btn_displayResults">
                <property name="text">
                 <string>Summarize Results</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QGroupBox" name="groupBox_2">
                <property name="title">
                 <string/>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_7">
                 <item>
                  <widget class="QRadioButton" name="radioButton_FFT">
                   <property name="text">
                    <string>FFT</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QRadioButton" name="radioButton_DFT">
                   <property name="text">
                    <string>DFT</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </item>
              <item row="4" column="4">
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Oversampling Ratio</string>
                </property>
               </widget>
              </item>
              <item row="4" column="2">
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_numIterations">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="6" column="1">
               <widget class="QLineEdit" name="lineEdit_interpolationCutoffDistance">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="6" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Interpolation Cutoff Distance (pixels)</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="Line" name="line_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Number of Iterations</string>
                </property>
                <property name="wordWrap">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="5" rowspan="7">
               <layout class="QGridLayout" name="gridLayout_3"/>
              </item>
              <item row="4" column="1">
               <widget class="QLineEdit" name="lineEdit_oversamplingRatio">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="btn_reconstruct">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>775</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Launch Reconstruction</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTextEdit" name="log">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1093</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Projection Calculator</string>
    </property>
    <addaction name="action_Create_Support"/>
   </widget>
   <widget class="QMenu" name="menuView_Volume">
    <property name="title">
     <string>View Volume</string>
    </property>
    <addaction name="action_Volume_Slicer"/>
   </widget>
   <addaction name="menuTools"/>
   <addaction name="menuView_Volume"/>
  </widget>
  <action name="action_Create_Support">
   <property name="text">
    <string>Launch Projection Calculator</string>
   </property>
  </action>
  <action name="action_Volume_Slicer">
   <property name="text">
    <string>Launch Volume Slicer</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEdit_pj</tabstop>
  <tabstop>lineEdit_support</tabstop>
  <tabstop>lineEdit_angle</tabstop>
  <tabstop>lineEdit_io</tabstop>
  <tabstop>lineEdit_numIterations</tabstop>
  <tabstop>lineEdit_oversamplingRatio</tabstop>
  <tabstop>lineEdit_interpolationCutoffDistance</tabstop>
  <tabstop>radioButton_on</tabstop>
  <tabstop>radioButton_off</tabstop>
  <tabstop>radioButton_extension</tabstop>
  <tabstop>btn_angles</tabstop>
  <tabstop>btn_io</tabstop>
  <tabstop>btn_projections</tabstop>
  <tabstop>btn_support</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
